src/main/java/uk/gov/justice/hmpps/prison/repository/InmateRepository.java

```diff
@@ -566,6 +566,18 @@ private List<OffenderCategorise> getCategorisationsWithValidAssessStatus(final L
         return offenderCategorisations.stream().filter(cat -> cat.getCategory() != null && validAssessStatus.contains(cat.getAssessStatus())).toList();
     }
 
+    public List<OffenderCategorise> getOffenderCategorisations(final List<Long> bookingIds, final String agencyId, final boolean latestOnly) {
+        final var dtos = jdbcTemplate.query(
+            InmateRepositorySql.GET_OFFENDER_CATEGORISATIONS.getSql(),
+            createParams("bookingIds", bookingIds,
+                "agencyId", agencyId,
+                "assessmentId", getCategoryAssessmentTypeId()),
+            OFFENDER_CATEGORY_MAPPER);
+
+        final var rawData = dtos.stream().map(OffenderCategoriseDto::toOffenderCategorise).toList();
+        return latestOnly ? removeEarlierCategorisations(rawData) : rawData;
+    }
+
```

Comment:
Just re-added all the old code and tests which supported the 2 reinstated endpoints